File: Kattis/Python/Stopwatch.py

def main():
    n = int(input())

    start = 0
    end = 0
    temp = 0

    t = bool(False)

    while n > 0:
        a = int(input())
        t = not t
        if t:
            start = a
        else:
            end = a
            temp += end - start
        n -= 1
    if(t):
        print("still running\n")
    else:
        print(temp, end="\n")

if __name__ == '__main__':
    main()

File: Kattis/C++/Stopwatch/Stopwatch.cpp
#include <bits/stdc++.h>

using namespace std;
typedef int64_t i64;

int main()
{
    i64 n, start = 0, end = 0;
    i64 temp = 0;

    cin >> n;

    bool t = false;

    while(n--)
    {
        i64 a;

        cin >> a;
        t = !t;

        if(t)
        {
            start = a;
        }
        else
        {
            end = a;
            temp += end - start;
        }
    }

    if(t) 
    {
        cout << "still running\n";
    }
    else
    {
        cout << temp << "\n";
    }
}